File: Unidad 3 Variables Aleatorias/Generacion Variables Aleatorias/Convolucion.py
# Descripción: Aproxima N(0,1) con suma de 12 números uniformes
import random
import tkinter as tk

def generar_convolucion():
    suma = sum(random.random() for _ in range(12))
    normal = suma - 6
    label_resultado.config(text=f"Variable por convolución: {normal:.4f}")

ventana = tk.Tk()
ventana.title("Convolución")
tk.Button(ventana, text="Generar", command=generar_convolucion).pack()
label_resultado = tk.Label(ventana, text="Variable por convolución: ?")
label_resultado.pack()
ventana.mainloop()

File: Unidad 3 Variables Aleatorias/Generacion Variables Aleatorias/Convolucion.java

// Descripción: Aproxima una N(0,1) sumando 12 variables uniformes
import java.util.Random;

public class Convolucion {
    public static void main(String[] args) {
        Random rand = new Random();
        double suma = 0;
        for (int i = 0; i < 12; i++) {
            suma += rand.nextDouble();
        }
        double normal = suma - 6;
        System.out.println("Variable por convolución: " + normal);
    }
}
